pkgs/development/libraries/rapidcheck/default.nix

```diff
@@ -0,0 +1,27 @@
+{ stdenv, cmake, fetchFromGitHub }:
+
+stdenv.mkDerivation rec{
+  name = "rapidcheck-${version}";
+  version = "git-20180927";
+
+  src = fetchFromGitHub {
+    owner = "emil-e";
+    repo  = "rapidcheck";
+    rev   = "de54478fa35c0d9cea14ec0c5c9dfae906da524c";
+    sha256 = "0n8l0mlq9xqmpkgcj5xicicd1my2cfwxg25zdy8347dqkl1ppgbs";
+  };
+
+  buildInputs = [ cmake ];
```

Comment:
`cmake` should be in `nativeBuildInputs`

Reply:
I wonder if tips like this could be automated �

Reply:
opened a borg issue here https://github.com/NixOS/ofborg/issues/245